src/pool.rs

```diff
@@ -314,7 +372,7 @@ impl ConnectionPool {
 
             match tokio::time::timeout(
                 tokio::time::Duration::from_millis(healthcheck_timeout),
-                server.query(";"),
+                server.query(";"), // Cheap query (query parser not used in PG)
```

Comment:
I vaguely recall this skips the planner not the parser.

Reply:
Makes way more sense!